.github/workflows/dotnet.yml

```diff
@@ -40,6 +40,9 @@ jobs:
     - name: Start containers
       run: docker compose -f docker-compose.yml up -d dev
 
+    - name: Run unit tests
+      run: dotnet test BTCPayServer.Plugins.Monero.UnitTests --verbosity normal
```

Comment:
tiny, but i think it should be moved before the Install Playwright browsers so if it fails there is no need to download docker images......